tests/v2/validation/auth/userretention/userretention_settings_test.go

```diff
@@ -0,0 +1,346 @@
+//go:build (validation || infra.any || cluster.any || sanity) && !stress && !extended
+
+package userretention
+
+import (
+	"testing"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/rancher/shepherd/clients/rancher"
+	"github.com/rancher/shepherd/pkg/session"
+	"github.com/sirupsen/logrus"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"github.com/stretchr/testify/suite"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+)
+
+type UserRetentionSettingsTestSuite struct {
+	suite.Suite
+	client        *rancher.Client
+	session       *session.Session
+	originalTTL   string
+	settingsReset bool
```

Comment:
We should keep the struct minimal, and I don't see the originalTTL and settingsReset fields being used anywhere. These fields can be removed.